library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -164,4 +172,8 @@ class ChuckerInterceptor @JvmOverloads constructor(
     companion object {
         private val UTF8 = Charset.forName("UTF-8")
     }
+
+    data class HttpRequestBody(
```

Comment:
Naming can be improved here:

```suggestion
    data class GraphQLRequestBody(
```

Reply:
Since the location of operation name was different depending on the library, it was absorbed using `JsonParser`

[apollo-android](https://github.com/apollographql/apollo-android/)  : 

```json
{
  "operationName": "xxx"
}
```

[retrofit-graphql](https://github.com/AniTrend/retrofit-graphql) :

```json
{
  "queryContainer" : {
       "operationName": "xxx"
   }
}
```